packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -7772,13 +7813,12 @@ describe('plugin-meetings', () => {
         it('sets correctly', () => {
           assert.notOk(meeting.permissionTokenPayload);
 
-          const permissionTokenPayloadData = {permission: {userPolicies: {a: true}}, exp: '1234'};
+          const permissionTokenPayloadData = {permission: {userPolicies: {a: true}}, exp: '123456'};
 
-          const jwtDecodeStub = sinon.stub(jwt, 'decode').returns(permissionTokenPayloadData);
-
-          meeting.setPermissionTokenPayload();
+          meeting.setPermissionTokenPayload(
+            'eyJhbGciOiJIUzI1NiJ9.eyJleHAiOiIxMjM0NTYiLCJwZXJtaXNzaW9uIjp7InVzZXJQb2xpY2llcyI6eyJhIjp0cnVlfX19.wkTk0Hp8sUlq2wi2nP4-Ym4Xb7aEUHzyXA1kzk6f0V0'
```

Comment:
```suggestion
            'permissionToken'
```

Reply:
This can't be taken Sreekanth. I had the same comment and got it clarified. Please go through the `setPermissionTokenPayload` method and you'll get answers for it. 

Earlier we were stubbing the old jwtDecode lib. Now with the new one, that is not easily possible. So we just send a valid-looking token for it to say it is a valid token. That's why this change is legit.